### PR TITLE
Myometrie update

### PR DESCRIPTION
**Pull request by Vigo28**
## Issue ticket number and link
https://github.com/ZuydUniversity/B2C6_B2A_Backend/issues/23

## Checklist before requesting a review
- [X] My code conforms to the code standards and definition of done (Document available on Teams)
- [X] I have performed a self-review of my code
- [X] My code is functional and runs without error
- [ ] This merging branch is up to date with the main branch

### CODE REVIEW

File: Backend_B2A.py [hunk 1]
```diff
@@ -5,16 +5,16 @@
 from dotenv import load_dotenv
 import base64
 import io
+from io import BytesIO
 import logging
+from reportlab.pdfgen import canvas
```
Comment (Hedgehog-From-Outerspace) [inline Backend_B2A.py:10]: Voeg nieuwe dependencies toe aan requirements.txt

File: Backend_B2A.py [hunk 2]
```diff
@@ -1153,70 +1159,99 @@ def download_patient_pdf(patient_id):
...
+        return send_file(
+            response,
+            as_attachment=True,
+            download_name=f'result_{result_id}_data.pdf',
+            mimetype='application/pdf'
+        )
 
     except Exception as e:
-        return jsonify({"message": "An error occurred"})
+        print(f"Fout opgetreden: {str(e)}")
```
Comment (Hedgehog-From-Outerspace) [inline Backend_B2A.py:1253]: Haal print() comment weg voor productie

**Comment by Hedgehog-From-Outerspace** [comment]
Geen issue nr + link
Geen unit testen (schrijf er 1tje ofzo)

**Comment by DevanStrife** [comment]
Branch is ook out of date met main. Als je niet weet hoe je dat fixed let me know en dan help ik je

File: Backend_B2A.py (hunk 1 above)
Comment (Vigo28) [reply Backend_B2A.py:10]: zie pull request https://github.com/ZuydUniversity/B2C6_B2A_Backend/pull/24

File: Backend_B2A.py (hunk 2 above)
Comment (Vigo28) [reply Backend_B2A.py]: print() is eruit na nieuwe commit

**Review by DevanStrife — APPROVED** [review]
lgtm

**Pull request merged**